src/chrome/content/rules/Findlawimages.com.xml

```diff
@@ -1,5 +1,4 @@
 <ruleset name="Findlaw_Images">
-	<target host="findlawimages.com" />
```

Comment:
Please add a hint about the mismatch and the mixed content issues on the related `findlaw.com` page.

Reply:
Hi , I added as a comment non working host, it is that ok?